src/bin/nydusd/main.rs

```diff
@@ -248,7 +248,15 @@ fn prepare_commandline_options() -> Command {
             Arg::new("log-rotation-size")
                 .long("log-rotation-size")
                 .help("Specify log rotation size(MB), 0 to disable")
-                .default_value("0")
+                .default_value("100")
```

Comment:
We should change the default value of nydusd which is somewhat of a UI. Changing the usual behavior is not good.

Reply:
Got it.